app.js

```diff
@@ -24,7 +71,7 @@ app.use(frameguard({
 app.use(helmet.csp({
   directives:{
     scriptSrc: ["'self'","'unsafe-inline'","data:", "https://cdn.optimizely.com", "https://app.optimizely.com", "https://basket.mozilla.org", "https://basket-dev.allizom.org","https://*.shpg.org/", "https://www.google-analytics.com/"],
-    connectSrc:["'self'", "206878104.log.optimizely.com", "https://basket.mozilla.org/", "https://basket-dev.allizom.org"],
+    connectSrc:["'self'", "5629570640.log.optimizely.com", "https://basket.mozilla.org/", "https://basket-dev.allizom.org"],
```

Comment:
I created a test optimizely account for this, which had a different id, but I had to allow it.